ospf6d/ospf6_nssa.h

```diff
@@ -45,7 +45,7 @@ extern unsigned char config_debug_ospf6_nssa;
 #define OSPF6_LSA_APPROVED      0x08
 #define OSPF6_LSA_LOCAL_XLT     0x40
 
-#define OSPF6_ABR_TASK_DELAY    7
+#define OSPF6_ABR_TASK_DELAY    5
```

Comment:
Don't we need to use the same for ospfv2?

Reply:
Currently no impact, I have also verified graceful-restart topotest ospf_gr_topo1/test_ospf_gr_topo1.py.
Change may be only needed to keep timers consistent